code/datums/station_traits/negative_traits.dm

```diff
@@ -290,3 +290,93 @@
 	report_message = "Due to budget cuts, we have downsized your escape pods."
 	trait_to_give = STATION_TRAIT_SMALLER_PODS
 	blacklist = list(/datum/station_trait/luxury_escape_pods)
+
+///A special trait, used only for when a revolution was won in the previous round. See /obj/effect/mapping_helpers/revolution_trash for more information.
+/datum/station_trait/revolutionary_trashing
+	name = "Post-Revolutionary Fervor"
+	show_in_report = TRUE
+	report_message = "Your station was recently reclaimed from a revolutionary commune. We aren't cleaning up after them."
```

Comment:
```suggestion
	report_message = "Your station was recently reclaimed from a revolutionary commune. We couldn't clean up after them in time."
```

ehhhh the idea of nanotrasen being so evil that they just let the shift after them deal with it is alright, but i'd much rather the concept of something like this happening out of incompetence rather than some level of malice